domains/Discussions/Controllers/QuestionsUpdateController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class QuestionsUpdateController extends Controller
+{
+    private Question $questions;
+
+    public function __construct(Question $questions)
+    {
+        $this->questions = $questions;
+    }
+
+    public function __invoke(int $questionId, Request $request): Response
+    {
+        $question = $this->questions->newModelQuery()->findOrFail($questionId);
+
+        if ($request->user()->cannot('update', $question)) {
+            abort(Response::HTTP_FORBIDDEN);
+        }
```

Comment:
We can remove the responsibility of manually sending an error if we use the `authorize()` method, as in `$this->authorize()`.

Reply:
For some reason I thought this was available only in Laravel .... 🤷‍♂️